gradle-plugins/build.gradle

```diff
@@ -1,10 +1,9 @@
-apply plugin: 'java'
+// Setup integTests
+apply from: "${buildScriptDirPath}/integTest.gradle"
 
 dependencies {
-  compile gradleApi()
```

Comment:
NB: `compile gradleApi()` is applied automatically by the `java-gradle-plugin`

Reply:
Exactly. That is why i put that note in "integTest.gradle"